code/controllers/ExtensibleSearchAPI.php

```diff
@@ -93,22 +94,43 @@ public function toggleSuggestionApproved($request) {
 	 *	@URLparameter term <{SEARCH_TERM}> string
 	 *	@return JSON
 	 */
-
 	public function getSuggestions($request) {
 
-		if(Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions') && ($suggestions = $this->service->getSuggestions($request->getVar('term'), $request->getVar('page')))) {
+		if(Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions')) {
+			$suggestions = $this->service->getSuggestions($request->getVar('Search'), $request->getVar('page'));
```

Comment:
404 indicates that the resource (in this case the search suggestions method) is missing, not that it merely found no results. A 404 will also display in DevTools as an error, which is a red herring as the query has completed successfully.